src/lib/github/issueCommentEditAction.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * function that enables an issue comment to be editable, that is to say,
+ * provides the delete and edit(overwrite) options.
+ *
+ * @param: context & issueCommentArray
+ * @returns: Promise<void>
+ */
+
+import { getBasicRepoProps } from "./utils";
+import { Context } from "./types";
+
+type CommentEditAction = "Delete" | "Edit";
+
+const Delete: CommentEditAction = "Delete";
+const Edit: CommentEditAction = "Edit";
+
+interface CommentEdit {
+  GhIssueId: number;
+  GhCommentId: number;
+  action: CommentEditAction;
+  commentsText: string[];
+}
+
+type CommentsEdit = CommentEdit[];
+
+export default async function issueCommentToEdit (context: Context, issueComment: CommentsEdit): Promise<void> {
```

Comment:
Make the function definition shorter by writing it on multiple lines:
```
export default async function issueCommentToEdit (
    context: Context,
    issueComment: CommentsEdit
): Promise<void> {
```
ideally code lines shouldn't go beyond 100 columns in length.